composer.json

```diff
@@ -31,6 +31,6 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.0"
+        "phpunit/phpunit": "^8.0"
```

Comment:
Why not `^9.5`?

Reply:
Because we have more important PHP things to do.

Reply:
It's just a matter of version, everything else remains the same. Also, you've just switched from 5 to 8, no major BC breaks between PHPUnit 8 and 9. :)

Reply:
> more important

I think **everything** is important, including PHPUnit. PHPUnit >= 9 has a better support for PHP 8.1 that we gonna migrate to soon.

Reply:
You are right, fixed in 03ab4eddd66c7623b9b9620ff82d4e4b7fa0972c